packages/actionmenu/src/react/item.tsx

```diff
@@ -12,7 +12,7 @@ import React, {
 } from 'react'
 
 import stylesheet from '../css'
-import { tagName } from '../vars'
+import { tagName as tagNames } from '../vars'
```

Comment:
Should we just rename this tagNames

Reply:
nbd, but imo, yes. I avoided just so that I didn't have to bump the major version. :)